Freya Assignment jQuery/app.js

```diff
@@ -0,0 +1,36 @@
+$( document ).ready(function() {
+    alert('Click to browse our awesome page!');
+});
+
+$('.readmore').click(function() {
+	$('#show-this-on-click').slideDown();
+	$('.readmore').hide();
+	$('.hide').show();
+});
+
+$('.hide').click(function(){
```

Comment:
Everytime I click on these buttons, it brings me up to the top of the page, this is because we are clicking on an anchor tag with a href="#".  You need to preventDefault on these clicks as well to prevent that behaviour.